### DISCUSSION TITLE
Question about concatenation

### DISCUSSION THREAD

**Discussion by Ikramkaabi** [solid-de/learn-java#5] [Week1 + Week2]
When we do the concatenation ( + ) with the type string we get a phrase or a one word ,like : 
System.out.pintln("concat" + "enation");
The result will be : concatenation
Moreover,  the concatenation with the type int will do the addition of the numbers. So I want to know if it possible what the concatenation give in java when we concat two variables with the type boolean ???

**Comment by Ikramkaabi** [comment]
If I have a grammatical or informational error, please correct it for me.  Thank you

**Reply by sihemnsibi** [reply]
la concaténation des variables booléens est interdit en java ( j'ai essayée de faire un exemple et cela provoque un erreur de compilation  car l'operateur + est indéfini pour le type booléen )

**Comment by IbrahimGharsalli** [comment]
In Java, you cannot concatenate two variables of type boolean directly. The boolean type can only have the values true or false, and it cannot be concatenated with other types of data like strings or integers.

One way to concatenate two boolean variables in Java is to convert them to strings using the String.valueOf() method and then concatenate the strings using the + operator. Here's an example:
boolean var1 = true;
boolean var2 = false;
String result = String.valueOf(var1) + String.valueOf(var2);
System.out.println(result);
